src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelCreationService.java

```diff
@@ -165,65 +137,78 @@ private void createCell(XSSFRow row, int cellIndex, String value, CellStyle styl
 
     private void initializeData(XSSFWorkbook workbook, XSSFSheet sheet, List<?> multipleCollection,
                                 List<String> subMultipleCollection, String leadCaseString) {
+        String leadCase = MultiplesHelper.getCurrentLead(leadCaseString);
+        log.info("Creating lead case EXCEL STRING: " + leadCaseString);
+        log.info("Creating lead case EXCEL: " + leadCase);
+
+        if (multipleCollection.isEmpty()) {
+            return;
+        }
+
+        var isStringRefsList = multipleCollection.get(0) instanceof String;
+        log.info(isStringRefsList ? "Initializing multipleRefs" : "Initializing data");
+
+        var orderedAllCasesList = MultiplesHelper.createCollectionOrderedByCaseRef(multipleCollection);
+        if (orderedAllCasesList.isEmpty()) {
+            return;
+        }
 
+        final int[] rowIndex = {1};
+        orderedAllCasesList.forEach((String caseYear, Map<String, Object> caseYearList) ->
+            caseYearList.forEach((String caseNum, Object caseItem) -> {
+                var multipleObject = isStringRefsList ? null : (MultipleObject) caseItem;
+                var ethosCaseRef = isStringRefsList ? (String) caseItem : multipleObject.getEthosCaseRef();
+
+                constructCaseExcelRow(workbook, sheet, rowIndex[0], ethosCaseRef, leadCase, multipleObject,
+                        !subMultipleCollection.isEmpty());
+                rowIndex[0]++;
+            }
+        ));
+    }
+
+    private void constructCaseExcelRow(XSSFWorkbook workbook, XSSFSheet sheet, int rowIndex, String ethosCaseRef,
+                                       String leadCase, MultipleObject multipleObject, boolean hasSubMultiples) {
         CellStyle styleForUnLocking = getStyleForUnLocking(workbook);
         CellStyle styleForLocking = getStyleForLocking(workbook, false);
         CellStyle styleForLockingLead = getStyleForLocking(workbook, true);
-        String leadCase = MultiplesHelper.getCurrentLead(leadCaseString);
-        log.info("Creating lead case EXCEL STRING: " + leadCaseString);
-        log.info("Creating lead case EXCEL: " + leadCase);
+        XSSFRow row = sheet.createRow(rowIndex);
+        int columnIndex = 1;
+
+        if (ethosCaseRef.equals(leadCase)) {
+            log.info("Lead: " + leadCase);
+            createCell(row, columnIndex, ethosCaseRef, styleForLockingLead);
```

Comment:
Can you create styleForLookingLead within this if statement rather than above. Otherwise it seems the variable could get set but never used